tf-modules/cumulus/variables.tf

```diff
@@ -487,3 +487,9 @@ variable "non_complete_execution_payload_timeout" {
   type        = number
   default     = 30
 }
+
+variable "archive_api_url" {
```

Comment:
There should be an associated CHANGELOG entry for this. Also, would you mind adding one for the change to the default `urs_url` variables that you made?

Reply:
:+1: sounds good to me, will do

Reply:
4bc0855